lib/facility.rb

```diff
@@ -1,14 +1,62 @@
 class Facility
-  attr_reader :name, :address, :phone, :services
+  attr_reader :name, :address, :phone, :services, :registered_vehicles, :hours, :address_id
+  attr_accessor :collected_fees
```

Comment:
because you are manipulating the collected fees from within this class, I don't think this needs to be an attr_accessor.